packages/framework-provider-local/src/library/searcher-adapter.ts

```diff
@@ -1,4 +1,4 @@
-import { Filter } from '@boostercloud/framework-types'
+import { FilterOld } from '@boostercloud/framework-types'
```

Comment:
Why renaming this type as `FilterOld`?

Reply:
I think he did that to avoid the need of changing all the providers at once to the new filters. So he left the old filters in place and used only the new ones in AWS

Reply:
🤔 and that's why we need to move the core features to separate rockets too...